### PR TITLE
WIP: docs: Module 5 Starport - code sample integration 2 of 3 (B9lab updates 4-4)

### PR DESCRIPTION
**Pull request by coldice**
Docs content update for Module 5, Section _Starport_ - part of the [b9lab-content](https://github.com/cosmos/sdk-tutorials/tree/master/b9lab-content).

This PR adds code samples to the content. The code shown here is taken from the [code example repo](https://github.com/cosmos/b9-checkers-academy-draft/) and has been hardly adjusted. The repo has been reviewed so no need to review it here. What we are looking for is to make sure that the introductory texts are not misleading or even downright wrong.

The Starport code sample integration PRs are split into three parts, also see #853 and #854

### Content state

**✔️ Completed**

### Technical review
- [x] requested
- [ ] completed

### Language review
- [ ] requested
- [ ] completed

### CODE REVIEW

File: b9lab-content/5-my-own-chain/03-starport-08-reject-game.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+A player cannot reject a game once they've played. However, as of now, you do not know whether a player played or not. To remediate this, you need to add a new field to the `StoredGame`. Let's call it `MoveCount`. So in `proto/checkers/stored_game.proto`:
+
+```proto [https://github.com/cosmos/b9-checkers-academy-draft/blob/329c6d0ae8c1dffa85cd437d0cebb246a827dfb2/proto/checkers/stored_game.proto#L15]
+storedGame := types.StoredGame{
+    ...
+    uint64 moveCount = 7;
+}
+```
+At this point, to have Protobuf recompile the relevant Go files, you may want to run:
```
Comment (toschdev) [inline b9lab-content/5-my-own-chain/03-starport-08-reject-game.md:44]: You can simplify this step by using

`starport generate proto-go`

File: b9lab-content/5-my-own-chain/03-starport-09-game-fifo.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    }
+    ```
+3. Finally, there needs to be an id that indicates _no game_. Let's use `"-1"`. In `x/checkers/types/keys.go `:
+    ```go [https://github.com/cosmos/b9-checkers-academy-draft/blob/2343af69cd1f2c22acfac13f46393aa8ce686685/x/checkers/types/keys.go#L32-L34]
+    const (
+        NoFifoIdKey = "-1"
+    )
+    ```
+
+To have Starport and Protobuf recompile these files, you can use:
```
Comment (toschdev) [inline b9lab-content/5-my-own-chain/03-starport-09-game-fifo.md:67]: If the intention is to build and compile, not to actually start running the chain, consider using

`starport chain build`
https://docs.starport.network/cli/#starport-chain-build

File: b9lab-content/5-my-own-chain/03-starport-10-game-deadline.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+```proto [https://github.com/cosmos/b9-checkers-academy-draft/blob/0d0e36a8ac86cddc457610856ddaab1b356cee84/proto/checkers/stored_game.proto#L18]
+message StoredGame {
+    ...
+    string deadline = 10;
+}
+```
+To have Starport and Protobuf recompile this file, you can use:
+
+```sh
+$ starport chain serve
```
Comment (toschdev) [inline b9lab-content/5-my-own-chain/03-starport-10-game-deadline.md:24]: Same as above, `starport chain build` might be more appropriate here

File: b9lab-content/5-my-own-chain/03-starport-08-reject-game.md (hunk 1 above)
Comment (xavierlepretre) [reply b9lab-content/5-my-own-chain/03-starport-08-reject-game.md:44]: Thank you.

**Comment by coldice** [comment]
Merged in b9lab-content-updates-rc1 (no diff to this branch)

**Pull request closed without merging**